### DISCUSSION TITLE
filter in search such as by date, comments or upvoting etc. would be very usefull.

### DISCUSSION THREAD

**Discussion by Kartik-Chaurasiya** [dailydotdev/daily#481] [Feature Requests]
filter in search such as by date, comments or upvoting etc. would be very usefull.

**Comment by idoshamun** [comment]
Hi!

Thanks for suggesting.
We have so many people requesting it and we want to address it in the future for sure.
Can't guarantee any time frame for now.
Let's keep the convo here: https://github.com/dailydotdev/daily/discussions/294
